governance-exercises/Poll Exercise 03.md

```diff
@@ -0,0 +1,110 @@
+
+> One can somewhat see the Instructions coming as a Governance Poll outcome from Maker as the “code” and the trust/trustee/3rd parties as the “machine” that safely executes the code and puts relevant data back to Maker. ...**These instruction sets are set out to be as detailed as possible** - including **clear specification** of checks and validations required along the way.
+>
+> Maker can structure instruction sets in any manner it prefers and submit via Governance Poll to the Trust - and given it is legal and possible - it will execute. ...
+>
+> As a safety measure we have also developed a library of key instruction sets that address key situations and can be put up as Governance Polls, **that we know are appropriately designed and confirmed executables.** This way most eventualities are planned for and can be acted upon confidently and speedily by Maker. ...
+>
+> **Trustee has no discretion for executions outside of Governance Poll approved instruction sets.** [emphasis added]
+
+There are three problems related to the Standing Orders in the Exercise Prompt, which run counter to both aspects of my mandate as a Governance Facilitator.
+
+First, the proposed Standing Order does not offer voters enough information to make an informed decision; this is due to the ambiguities in the proposed Standing Order, which I assume are non-trivial. If the Standing Order/Instruction Set is to be akin to safely executable code, it cannot be ambiguous without presenting significant risk to Maker. 
+
+For the Clydesdale RWA structure, a library of standing orders/instruction sets was proactively developed and validated by the pertinent Maker ecosystem actors prior to any immediate need. In the immediate situation, we are apparently missing the expert input/assessment of such ecosystem actors, including the Strategic Finance Core Unit. (SFCU's [mandate](https://mips.makerdao.com/mips/details/MIP55c3SP12#sentence-summary) is the monitoring of RWA vaults and includes the review of legal structures, resolutions, wordings and agreements.) From the Exercise prompt, I assume this oversight may be due to Emilia's mistaken belief that her conversations with "large MKR governance holders" sufficed, without further ado, to bring the proposal forward. (See [here](https://forum.makerdao.com/t/proposal-to-increase-debt-ceiling-for-mip65-by-750m-to-1250m/20060#makerdao-resolution-instruction-sets-12) for another pertinent Clydesdale precedent, announcing that a proposed instruction set will be shared along *with* an "assessment by a relevant member with legal credentials in the Maker Community.")
```

Comment:
Essentially, yes. Though note in this case SFCU are assumed to not exist, and have been replaced by Faraz's advisory teams.